pkg/virt-api/validating-webhook/validating-webhook.go

```diff
@@ -321,6 +321,25 @@ func validateVirtualMachineSpec(field *k8sfield.Path, spec *v1.VirtualMachineSpe
 		return causes
 	}
 
+	// Validate memory size if values are not negative
+	if spec.Domain.Resources.Requests.Memory().Value() < 0 {
```

Comment:
I was thinking to prepare one function for checking quantity for memory insted of ```if ... else if...``` although I was following code style from volume/disks checks which is simple. Should I change it? I think it might be worth to refactor to one func for negative values when cpu quantity will be here as well.

Reply:
It's very easy to follow like you did it. Good enough for now.

Reply:
Since you mentioned CPU, we will need validation for it too. Maybe in a follow-up PR? ;)

Reply:
Yeah I think I can do that, if this PR is good enough to merge then I will have to do something new so I can check CPU as well and improve validation where is needed.